tensorflow_datasets/image/fgvc_aircraft.py

```diff
@@ -0,0 +1,151 @@
+"""Fine-Grained Visual Classification of Aircraft (FGVC-Aircraft) is a benchmark fgvc-aircraft-2013b/dataset for the fine grained visual categorization of aircraft."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import PIL
```

Comment:
Do not import PIL. Use OpenCV through tfds.core.lazy_imports.cv2 instead.

You can find an example here: https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/image/cbis_ddsm.py